perfkitbenchmarker/scripts/database_scripts/launch_driver.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script drives launch_my_sql service which is a wrapper class for
+# mysql_service_benchmark. 
+# Driver provisions a new instance approximately every three days.
+# The run phase is completed several times daily with results uploaded to
+# a designated Google Cloud Storage Bucket.
+
+# If desired, per second data graphs can be created and uploaded to a designated
+# Google Cloud Storage Bucket.
+
+# The values below are provisioned for realistic benchmarking.
+
+# This script is designed for benchmarking purposes. Therefore, a value for
+# a Google Cloud Storage Bucket must be passed in when called.
+
+# Example call:
+#    ./perfkitbenchmarker/scripts/database_scripts/launch_driver.sh bucket_name
+
+# To utilize per second data graphs, include a storage bucket as a second input.
```

Comment:
I removed this functionality down the line.. just forgot to update the doc strings!

Good catch!